cvat/apps/iam/rules/memberships.rego

```diff
@@ -92,11 +94,16 @@ allow {
     input.resource.role != organizations.OWNER
 }
 
+# member can leave the organization except cases when member is the owner or maintainer
 allow {
     input.scope == utils.DELETE
     input.resource.is_active
-    utils.is_sandbox
-    input.resource.role != organizations.OWNER
+    organizations.is_member
+    input.resource.organization.id == input.auth.organization.id
     input.resource.user.id == input.auth.user.id
+    not {
+        organizations.OWNER,
+        organizations.MAINTAINER
```

Comment:
@Marishka17 , why cannot maintainer leave the organization?

Reply:
@nmanovic, For me, maintainer should also be able to leave the organization. But we have [here](https://github.com/opencv/cvat/blob/783c414cfd08b7d0bc1510936bca955fa0c5a733/cvat/apps/iam/rules/memberships.rego#L80) and [here](https://github.com/opencv/cvat/blob/783c414cfd08b7d0bc1510936bca955fa0c5a733/cvat/apps/iam/rules/tests/configs/memberships.csv?plain=1#L9) such limitations..

Reply:
The idea behind the rule was a maintainer cannot delete another maintainer. Maintainers should be possible to leave the organization for sure.